_includes/navbar.html

```diff
@@ -28,10 +28,10 @@
                         <ul class="dropdown-menu{% if forloop.last %} dropdown-menu-right{% endif %}" aria-labelledby="drop{{item.index}}">
                             {% for entry in item.subfolderitems %}
                                 <li>
-                                    <a class="nav-link" href="/{{ entry.url }}">{{ entry.page }}</a>
+                                    <a class="dropdown-item" href="/{{ entry.url }}">{{ entry.page }}</a>
```

Comment:
Class adjusted to match bootstrap documentation: https://getbootstrap.com/docs/4.0/components/navbar/#nav